MANIFEST.in

```diff
@@ -7,41 +7,38 @@
 
 include .dockerignore
 include .editorconfig
+include .env.example
 include .tx/config
+include *.json
+include *.rst
+include *.sh
+include *.txt
+include *.yaml
+include *.yml
 include docker/haproxy/Dockerfile
 include docker/nginx/Dockerfile
 include docker/postgres/Dockerfile
 include Dockerfile
+include LICENSE
+include pytest.ini
 include scripts/bootstrap
 include scripts/deploy
 include scripts/server
 prune docs/_build
-recursive-include kpiit *.po *.pot *.mo
 recursive-include docker *.cfg *.conf *.crt *.ini *.key *.pem *.sh
-
-# added by check_manifest.py
-include *.rst
-include *.sh
-include *.txt
-include *.yml
-include LICENSE
-include pytest.ini
 recursive-include docs *.bat
+recursive-include docs *.md
 recursive-include docs *.py
 recursive-include docs *.rst
 recursive-include docs *.txt
 recursive-include docs Makefile
-recursive-include tests *.py
-
-recursive-include tests *.json
-recursive-include docs *.md
-recursive-include tests *.html
-include .env.example
-
-include *.yaml
-recursive-include scripts *.py
-recursive-include scripts *.sh
-include *.json
 recursive-include kpiit *.cfg
+recursive-include kpiit *.po *.pot *.mo
+recursive-include openshift *.conf
 recursive-include openshift *.yaml
 recursive-include openshift deploy
+recursive-include scripts *.py
+recursive-include scripts *.sh
+recursive-include tests *.html
+recursive-include tests *.json
+recursive-include tests *.py
```

Comment:
did you go one by one or do you have something that autocompletes the MANIFEST? :)

Reply:
``check-manifest -u`` does the job :-)